client/components/Guarentee.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Guarentee = () => (
```

Comment:
Are you planning on putting more information into your dataset for the Guarantee component